### PR TITLE
MAINT: Remove deprecated events and methods.

### PR DESCRIPTION
**Pull request by Carreau**
select and deselect events were supposed to be removed in 0.4.9, and two functions were supposed to be private in 0.4.11.

Remove them.

### CODE REVIEW

**Comment by Carreau** [comment]
@tlambert03 you seem to be the one that both introduced the deprecation/removal and the private functions that still use those events. I seem to have a hard time following where this is used and how the code should be updated.

**Comment by tlambert03** [comment]
looks like they are still used here ... and until someone updates the points layer, it probably can't be removed:
https://github.com/napari/napari/blob/22cd5596838a98d402ce03ae0f7031d91bcf9bc2/napari/layers/base/base.py#L1759-L1769

**Review by Czaki — APPROVED** [review]

**Comment by kcpevey** [comment]
@Carreau any updates here? Looks like you have one approval and are passing tests.

**Review by Czaki — APPROVED** [review]

File: napari/layers/base/base.py
Comment (Czaki) [inline napari/layers/base/base.py]: It is required now

Comment (Carreau) [reply napari/layers/base/base.py]: > It is required now

Readded, as well as the warning imports.

**Comment by Carreau** [comment]
I just want to note that this PR has been opened for 9 month now. And it's becoming slightly annoying to have to rebase/fix because a PR that was opened more than 7 month after this one was approved a couple of time was merged.

**Comment by Czaki** [comment]
A few days ago, we start working about update of responsibilities (ex code ownership) and assignments persons to PR. 

Maybe you know some application that allows to create a task queue (like revisiting this PR after 72h - then it will appear on the todo list after this time)?

**Comment by Carreau** [comment]
> A few days ago, we start working about update of responsibilities (ex code ownership) and assignments persons to PR.

Thanks, I've seen an increase of nudge reviews, so your efforts are seen. 

> Maybe you know some application that allows to create a task queue (like revisiting this PR after 72h - then it will appear on the todo list after this time)?

Not from the top of my head. I tend to use the "last updated" on GitHub view, and find that it's not a problem if you try to keep the number of PRs relatively small and close dormant ones.

Would it help you to make a cron workflow that does that with a say "nudge" label ? 

```python
import requests
from datetime import datetime, timedelta

owner = "napari"
repo = "napari"
#token = ... Your GitHub personal access token

label_to_check = "nudge"

headers = {"Authorization": f"token {token}"}

# Get all open pull requests
url = f"https://api.github.com/repos/{owner}/{repo}/pulls?state=open"
response = requests.get(url, headers=headers)
pulls = response.json()

for pull in pulls:
    labels = [label["name"] for label in pull["labels"]]

    url = f"https://api.github.com/repos/{owner}/{repo}/pulls/{pull_number}"

    response = requests.get(url, headers=headers)
    pull = response.json()

    labels = [label["name"] for label in pull["labels"]]
    if label_to_check in labels:
        continue

    url = f"https://api.github.com/repos/{owner}/{repo}/issues/{pull['number']}/events"
    response = requests.get(url, headers=headers)
    events = response.json()

    label_events = [event for event in events if event["event"] in "unlabeled" and event["label"]["name"] == label_to_check]

    if label_events:
        # check if label has been remove 72+ hours ago if so re-add
        # we may want to check for comments from napari (core) dev in
        # the last 72h as well and add/remove label as necessary ?
        last_label_event = label_events[-1]
        label_removed_at = datetime.strptime(last_label_event["created_at"], "%Y-%m-%dT%H:%M:%SZ")
        if datetime.now() - label_removed_at > timedelta(hours=72):
            url = f"https://api.github.com/repos/{owner}/{repo}/issues/{pull['number']}/labels"
            response = requests.post(url, headers=headers, json={"labels": [label_to_check]})
```

**Comment by Czaki** [comment]
I have a plan to investigate writing own GitHub app that one of ability will be to remind one after a given time. But the investigation of this could take time. Maybe setup a server with corn could be a good idea.

**Comment by Carreau** [comment]
I do have https://github.com/MeeseeksBox/meeseeksdev hosted on heroku, feel free to send PR to extend it. Though I'm not sure it does cron.

**Comment by Czaki** [comment]
Did you pay for Heroku, or free tier is still available?

**Comment by Carreau** [comment]
> Did you pay for Heroku, or free tier is still available?

It's a paid account. ~$7 per month.

**Comment by Carreau** [comment]
thanks !

**Pull request merged**
